modules/media/help/media.md

```diff
@@ -1,51 +1,43 @@
-# Media Module
+# Media
 
-## Overview
+The Media module serves as a repository for files associated with a particular 
+candidate-timepoint or instrument in a study.  Users can access and upload files 
+such as PDF scans, recordings, log files, or stimulus presentation media in multiple 
+formats such as .pdf, .mp4, .mp3, .txt, etc. 
 
-Media module allows users to **upload**, **search** and **edit** media files associated with a specific candidate timepoint in Loris.
-Any kind of data associated with a candidate timepoint can be uploaded through this module: PDFs, videos, recordings, scripts, log files, etc. Files can optionally be associated to a specific instrument form within a given candidate timepoint.
+## Searching for a Media File
 
+Under the Browse tab, use the Selection Filters to search for files by fields such 
+as including filename, file type, candidate PSCID, Visit Label, Instrument, Site, or 
+name of user who uploaded the file.  Partial string matching can be used on many 
+fields (e.g. filename).  As filters are selected, the below data table will 
```

Comment:
..., the data table on the lower section of the screen will ... maybe?

Reply:
@zaliqarosli @PapillonMcGill you will have to argue with @leighevans it's not my frenglish here

Reply:
If this is the only request for change, I think it can be altered and ready to go!

Reply:
as currently written, ... the below data table ... mean the table that is below the data which of course is not appropriate. @ridz1208 Could you check with @leighevans in person?

Reply:
@PapillonMcGill yes, I agree. I was just trying to get @ridz1208 to pay attention ;)

Reply:
@ridz1208 @PapillonMcGill @leighevans @HenriRabalais made and pushed this change